### PR TITLE
NWO to only use Fabric Binaries

### PR DESCRIPTION
**Pull request by davidkel**
follow-on from #423 as merge mess meant I needed to open a new PR
Signed-off-by: Dave Kelsey <d_kelsey@uk.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,65 +80,67 @@ orion-server-images:
...
 .PHONY: integration-tests-iouorionbe
-integration-tests-iouorionbe: orion-server-images
-	cd ./integration/fabric/iouorionbe; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests-iouorionbe: orion-server-images download-fabric
+	cd ./integration/fabric/iouorionbe; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) .
 
 .PHONY: integration-tests-atsacc
-integration-tests-atsacc:
-	cd ./integration/fabric/atsa/chaincode; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests-atsacc: download-fabric
```
Comment (adecaro) [inline Makefile:110]: do we still need this dependency target here?
The CI downloads the binary before starting the tests and in the README you already tell that the binaries must be downloaded. No?

**Review by adecaro — COMMENTED** [review]
LGTM. Just a clarification and I'll approve :)

File: Makefile (hunk 1 above)
Comment (davidkel) [reply Makefile:110]: Thanks @adecaro looks like I missed one when I removed downloading fabric from the targets per previous comment, I will remove it.

**Review by adecaro — APPROVED** [review]
LGTM

**Pull request merged**